src/qml/qgismobileapp.qml

```diff
@@ -322,6 +322,42 @@ ApplicationWindow {
       }
     }
 
+    DragHandler {
+      id: rotateDragHandler
+      enabled: rotateFeaturesToolbar.rotateFeaturesRequested == true && !freehandButton.visible && !freehandButton.freehandDigitizing
```

Comment:
Hm, here I don't think we should care about freehand button -- if we have requested a feature rotation, we wouldn't be freehand digitizing anyways.